src/styles/_stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  line-height: 16px;
+  width: 100%;
+  margin-top: 16px;
+
+  &__list {
+    @include center;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    justify-content: space-between;
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+  }
+
+  &__star:nth-child(n + 5) {
+    background-image: url('../images/star.svg');
+  }
+
+  &__reviews {
+    @include roboto400;
+
+    justify-content: space-between;
```

Comment:
The `justify-content` property is used here, but it will only have an effect if the `.stars__reviews` element is a flex container. Ensure that the `.stars__reviews` class has `display: flex;` applied if you intend to use `justify-content`.